src/index.js

```diff
@@ -66,102 +59,234 @@ class Radar {
     Cookie.setCookie(Cookie.DESCRIPTION, description);
   }
 
-  static trackOnce(callback) {
-    const publishableKey = Cookie.getCookie(Cookie.PUBLISHABLE_KEY);
-
-    if (!publishableKey) {
-      if (callback) {
-        callback(STATUS.ERROR_PUBLISHABLE_KEY);
-      }
+  static getLocation(callback) {
+    if (!callback) {
```

Comment:
We should probably throw an error if there's no callback. Otherwise it wont be clear why nothing is happening.